ProjFS.Mac/PrjFSKextTests/KextMockUtilities.hpp

```diff
@@ -111,6 +111,11 @@ class MockCalls
 
     static void Clear();
     
+    static bool DidCallAnyFunctions()
```

Comment:
I think this is fine for now, but in general I'm not sure how I feel about it.
basically we ensure nothing that would record a mock is called.  To be more explicit would

    XCTAssertFalse(MockCalls::DidCallFunction(KextMessageLogged));

Work?

Reply:
> we ensure nothing that would record a mock is called

Yep that's correct.  The idea here was that we don't expect the vnode cache code to be hitting _any_ of our mocked functions.

> To be more explicit would ...<snippet>... Work?

This would only validate that the vnode cache didn't log anything, which is more permissive than the current check.

You did raise a great point offline that we don't currently have a way to validating "DidOnlyCallFunction" (i.e. that only the specified functions were called).  I think this would be useful as well, but I'll hold off on adding it until we get more mileage with `DidCallFunction` to determine how useful (or not) it is.